mailserver/registry/verifier.go

```diff
@@ -0,0 +1,34 @@
+package registry
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/p2p/discover"
+)
+
+type RegistryVerifier struct {
+	rc         *RegistryCaller
+	pendingOpt bool
```

Comment:
so pending allows to include pending block as a source of info when call is made? is it useful? i would use generated code directly

Reply:
@dshulyak it's only use for testing purpose. When using the simulated backend after deploying the contract we can call it only with `&bind.CallOpts{Pending: true}`

Reply:
thats definitely not the case
did you commit on backend after deploying the contract?

Reply:
i checked the code. you should use SimulatedBackend without interface. it will allow you to commit a block when necessary. If you will add it after contract is deployed and when node is added - everything will work with this pending option